services/oldsService.js

```diff
@@ -265,7 +267,47 @@ const updateRecord = async (req, res) => {
         console.log(`  Success: ${changed}`)
 };
 
+
+
+// V2
+
+const who = async (req) => {
+    const regx = new RegExp("(?<=Bearer )[a-fA-F0-9]+")
+    const token = regx.exec(req.get("Authorization"))[0];
+
+    if ( typeof token === 'undefined') {
+        return {};
+        }
```

Comment:
This one will crash if we don't have auth in request.
The safer usage would be:

```
const regresult = regx.exec(req.get("Authorization"))
if (! regresult) { ... }
const token = regresult[0];
```